tests/x11/multi_users_dm.pm

```diff
@@ -20,15 +20,15 @@ use testapi;
 use utils;
 
 sub ensure_multi_user_target {
-    systemctl 'isolate multi-user.target';
+    type_string "systemctl isolate multi-user.target\n";
```

Comment:
I don't understand, why is this necessary?

Reply:
That's the reason why I've put the label. But this changed the behavior, as with assert_script_run command fails, even though command worked and we see login screen. So I will take a closer look why we don't get anything on the serial.